### DISCUSSION TITLE
Difference between the Docker Tags?

### DISCUSSION THREAD

**Discussion by digitalbarrito** [haugene/docker-transmission-openvpn#2530] [Q&A]
Curious what the difference between the tags [here](https://hub.docker.com/r/haugene/transmission-openvpn/tags) are?

What's the difference between beta, dev, kinetic, focal, and latest?

Interested in getting Transmission 4, and potentially the updates to the Flood UI, so I'm wondering if I should use a different docker image? I'm currently using "latest"

**Comment by pkishino** [comment]
transmission 4 is currently still on the :beta tag
